multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -18,6 +20,28 @@
 automm_presets = Registry("automm_presets")
 matcher_presets = Registry("matcher_presets")
 
+default_hyperparameter_tune_kwargs = {
+    "searcher": "bayes",
+    "scheduler": "ASHA",
+    "num_trials": 512,
+}
+
+default_tunable_hyperparameters = {
+    "optimization.learning_rate": tune.loguniform(1e-5, 1e-2),
+    "optimization.optim_type": tune.choice(["adamw", "sgd"]),
+    "optimization.max_epochs": tune.choice(list(range(5, 31))),
```

Comment:
@FANGAreNotGnu I'm not sure if this default setting works for detection, since detection usually requires more training epochs. We might need to override these values in detection_hpo presets.

Reply:
Each problem type can update the keys with its specific values. This just provides the default.